src/Query/LaravelQuery.php

```diff
@@ -503,15 +518,24 @@ public function createBulkResourceforResourceSet(
         ResourceSet $sourceResourceSet,
         array $data
     ) {
+        $verbName = 'bulkCreate';
+        $mapping = $this->getOptionalVerbMapping($sourceResourceSet, $verbName);
+
         $result = [];
         try {
             DB::beginTransaction();
-            foreach ($data as $newItem) {
-                $raw = $this->createResourceforResourceSet($sourceResourceSet, null, $newItem);
-                if (null === $raw) {
-                    throw new \Exception('Bulk model creation failed');
+            if (null === $mapping) {
```

Comment:
Drop through to default multiple-run-of-singleton handler for bulk update/create